test/e2e/apibinding/apibinding_permissionclaims_test.go

```diff
@@ -291,3 +290,128 @@ func bindConsumerToProvider(ctx context.Context, t *testing.T, consumerWorkspace
 	require.NoError(t, err, "error retrieving consumer workspace %q API discovery", consumerWorkspace)
 	require.True(t, resourceExists(resources, "cowboys"), "consumer workspace %q discovery is missing cowboys resource", consumerWorkspace)
 }
+
+func TestAPIBindingPermissionClaimsVerbs(t *testing.T) {
```

Comment:
Just for sake of sanity - can we extend this test to have mixed permissions:
1. I can create configMap to start with, but not a secret
2. Updated Export allows creating secrets now too
3. User removed permission - Export should still function, just not with full permissions. Warning raised on binding?

Basically, checking updates on both ends makes a difference, and mixed permissions work. 

Wdyt? Not overkill?

Reply:
Makes sense to me! I'll push an update tomorrow.

Reply:
@mjudeikis Addressed and PR is ready for another review. 🙂